client/src/main/kotlin/net/syncthing/java/client/Main.kt

```diff
@@ -13,6 +13,13 @@
  */
 package net.syncthing.java.client
 
+// this is archived
+//
+// I'm not sure if anyone still needs it. If someone needs it,
+// it should be moved to an other package because the SqlRepository
+// should not be a dependency of this module.
+
+/*
```

Comment:
I think this is mainly intended for testing. Using a command line tool for that is much easier than compiling an Android app.